src/atc/orchestrators/eh2bronze/EhToDeltaBronzeTransformer.py

```diff
@@ -0,0 +1,73 @@
+import pyspark.sql.functions as f
+from atc_tools.time import dt_utc
+from pyspark.sql import DataFrame
+
+from atc.etl import Transformer
+from atc.tables import TableHandle
+
+
+class EhToDeltaBronzeTransformer(Transformer):
+    """
+    This class transforms eventhub data into the following schema:
+
+    | BodyId | Body | EnqueuedTimestamp | StreamingTime | pdate |
+    |--------|------|-------------------|---------------|-------|
+    |...     |...   |...                |...            |...    |
+
+    Parameters:
+    target_dh: DeltaHandle for the target delta table (bronze)
+    df: A dataframe containing raw eventhub data
+
+    Returns:
+    A dataframe with the above mentioned schema
+
+
+    """
+
+    def __init__(self, target_dh: TableHandle):
+        super().__init__()
+        self.target_dh = target_dh
+        self._eh_cols = [
+            "BodyId",
+            "Body",
+            "EnqueuedTimestamp",
+            "StreamingTime",
+            "pdate",
+        ]
+
+    def process(self, df: DataFrame) -> DataFrame:
+        target_df = self.target_dh.read()
+
+        assert set(self._eh_cols).issubset(target_df.columns)
+
+        # Generate id for the eventhub rows using hashed body
+        df = df.withColumn(
+            "BodyId",
```

Comment:
as said above, please include the Enqueued Timestamp, and maybe even also the properties column to create this hash.